kubos/override/reimplemented_modules/access.py

```diff
@@ -0,0 +1,420 @@
+# Kubos SDK
+# Copyright (C) 2016 Kubos Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This module is an altered copy of the yotta/lib/access.py module.
+The kubos changes in here are specifically to change the yotta behavior
+around fetching modules from a registry. Specifically changes start on
+line 127
+'''
+
+# standard library modules, , ,
```

Comment:
They came with the original yotta: https://github.com/ARMmbed/yotta/blob/master/yotta/lib/access.py#L6